iphone/TitaniumKit/TitaniumKit/Sources/API/TiUtils.m

```diff
@@ -1420,6 +1420,51 @@ + (void)applyConstraintToView:(TiUIView *)view forProxy:(TiViewProxy *)proxy wit
   ApplyConstraintToViewWithBounds([proxy layoutProperties], view, bounds);
 }
 
++ (NSString *)composeAccessibilityIdentifier:(TiUIView *)view
+{
+  NSString *accessibilityLabel = view.accessibilityLabel;
```

Comment:
Will it always be unique? What is logic in this method to calculate accessibilityIdentifier.

Reply:
Not unique.
Logic is the same as for android `content description`: https://github.com/appcelerator/titanium_mobile/blob/af140652142b72b5e07eb5ed55964089ad2905c5/android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java#L2167